### PR TITLE
[WIP]expose the cluster-autoscaler logs to the user

### PR DESCRIPTION
**Pull request by vlvasilev**
**How to categorize this PR?**

/area logging
/kind enhancement
/priority normal

**What this PR does / why we need it**:
With this PR we expose the Cluster-autoscaler logs to the user.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Expose cluster-autoscaler logs to the user
```

### CODE REVIEW

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 1]
```diff
@@ -171,6 +171,11 @@ data:
         Parser              kubeapiserverParser
         Reserve_Data        True
 
+    [FILTER]
```
Comment (rfranzke) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Hm, we don't use this construct (`Record type user`) for other component logs that we expose to end-users (controller-manager, scheduler) - how does it work for them?

Comment (vlvasilev) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Hm. you are right, as I see now we use only predefined dashboards.

**Comment by vlvasilev** [comment]
Now I have add new dashboard but I didn't test it, because my local setup is broken and I have to leave for the moment.
When I test it I will remove [WIP] from the title.
Sorry for the inconvenience.

**Comment by vpnachev** [comment]
It looks very weird to have the logs of the cluster autoscaler in the VPA dashboard.
It would be better to have dedicated dashboard for the cluster autoscaler with some metrics and show the logs there (cc @wyb1 @istvanballok @prashanth26 @hardikdr ).

Another option is to simply move all logging related panes to the `Kubernetes Control Plane Status` dashboard - wdyt?

**Comment by Kristian-ZH** [comment]
I Do not think that adding this Panel to the `Kubernetes Dashboard Status` is a good idea because this Dashboard is really huge and uncomfortable for using.

We also can not make separate Dashboard Like `Pod Logs` which includes all Pods available for the users because when there is a select field, the selected option is applied to the URL and can be modified after that

So the best option for me is to create a separate Dashboard for each of the components that includes a basic metrics for it and its logs. Also we should add a search field, which the users will be able to use for searching a certain message in their logs

**Comment by rfranzke** [comment]
What's the plan with this PR? Does it make sense to keep it open, or will you rather rework the dashboards etc. in a larger PR?

**Comment by Kristian-ZH** [comment]
I think that reworking the dashboards in a larger PR is a better option

**Comment by rfranzke** [comment]
Makes sense. Then let's
/close
this one in favor of https://github.com/gardener/gardener/issues/2677#issuecomment-696577387

**Pull request closed without merging**